core/error_macros.h

```diff
@@ -426,6 +427,7 @@ void _err_print_index_error(const char *p_function, const char *p_file, int p_li
 #define CRASH_NOW()                                                                  \
 	if (true) {                                                                      \
 		_err_print_error(FUNCTION_STR, __FILE__, __LINE__, "FATAL: Method failed."); \
+		_err_flush_stdout();                                                         \
```

Comment:
That makes sense, though it could have been better suited in a dedicated PR (and it should be fixed in `master` too).

Reply:
Ah yes this crept in because it was downsized from a larger PR, and doing the flush here was very obviously needed during development as otherwise it crashes before we see the error messages.

Can move this to a separate PR if desired, and will do one for 4.x as it will be good to fix there too.

Reply:
Have removed this now to go in separate PR.